src/Digdir.Domain.Dialogporten.Application/Externals/IOrganizationRegistry.cs

```diff
@@ -3,4 +3,11 @@ namespace Digdir.Domain.Dialogporten.Application.Externals;
 public interface IOrganizationRegistry
 {
     Task<string?> GetOrgShortName(string orgNumber, CancellationToken cancellationToken);
+    Task<OrganizationLongName[]> GetOrganizationLongNames(string orgNumber, CancellationToken cancellationToken);
 }
+
+public sealed class OrganizationLongName
+{
+    public required string LongName { get; init; }
+    public required string Language { get; init; }
```

Comment:
De har andre localizations enn det vi bruker afaik. "en", "nn", "no". Burde muligens konvertere her?

Reply:
Hm, yeah.
Vi kan ha en intern mapping i orgRegistryClient da?

Reply:
yeah. Verdiene er isåfall nb_NO, nn_NO og EN?

Reply:
Ah, hm, nvm. Vi støtter "no", "nb", "nn, "nb-NO", "nn-NO"
Break på IsValidCultureCode i CultureTypes.cs om du vil se hele lista

Reply:
Hva fant dere ut her? @arealmaas 
Mener å huske at `en`, `nn`, og `no` er gyldige?

Reply:
Jepp, de er det